File: src/main/java/com/twoguis/carfixer/model/Veiculo.java
package com.twoguis.carfixer.model;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Veiculo {
    private int id;
    private String placa;
    private String modelo;
    private Tipo tipo;
    private int id_cliente;
    private List<Agenda> agendas;
}

File: src/app/model/veiculo.ts
import { Agenda } from "./agenda";

export class Veiculo {
    id: number;
    placa: string;
    modelo: string;
    tipo: string;
    id_cliente: number;
    agendas: Agenda[]

    constructor(){
        this.id = 0;
        this.placa = "";
        this.modelo = "";
        this.tipo = "";
        this.id_cliente = 0;
        this.agendas = [];
    }
}
